app/controllers/IdentityController.scala

```diff
@@ -45,9 +46,8 @@ class IdentityController(
           InternalServerError
         },
         cookies => {
-          SafeLogger.info("Successfully set password")
-          // TODO: add cookies to response header
-          Ok
+          SafeLogger.info("Successfully set passwrod")
```

Comment:
SafeLogger.info("Successfully set password")

Reply:
Nice suggestion! I'll also add to the unsuccessful case

Reply:
I've also added more detailed logging to `submitMarketing`, seeing as I'm here

Reply:
@johnduffell (in case you're interested) logback supports storing meta data in a [thread-local variable](https://logback.qos.ch/manual/mdc.html). You can then configure a pattern in logback to automatically include this meta data. We started doing this in contributions-frontend but never quite got there. It's probably too much effort   / not a priority to introduce this into support-frontend, but I always liked the idea of it!

Reply:
thanks @Mullefa but would threadlocal work with scala execution contexts?  I remember @rupertbates and I independently implemented a very similar solution using implicits, but I seem to remember neither of us were 100% keen on the idea!